content/applications/inventory_and_mrp/inventory/routes/concepts/stock_warehouses.rst

```diff
@@ -1,47 +1,99 @@
+In Odoo, this can be done by using **Virtual Locations**.
 
-Let’s say you have two warehouses: Warehouse A and Warehouse B. Create a new warehouse, that will be
-a virtual one. It will allow you to take the stock from A or B. To do so, go to your inventory app
-settings and enable the multi-warehouses feature. Then, go to the warehouses menu and click on
-create.
+.. note::
+   To create virtual locations in warehouses, :guilabel:`Storage Locations` will need to be
+   enabled. To do so, go to :menuselection:`Inventory --> Configuration --> Settings`, scroll down
+   to the **Warehouse** section, click the checkbox next to :guilabel:`Storage Locations`, and
+   :guilabel:`Save` changes.
+
+Create and configure a virtual parent location
+==============================================
+
+Before creating any virtual stock locations, a new warehouse will need to be created that will act
+as a virtual warehouse, and will be the parent location of other physical warehouses.
+
+Create a new warehouse
+----------------------
 
-.. image:: stock_warehouses/ware1.png
+To create a new warehouse, go to :menuselection:`Inventory --> Configuration --> Warehouses`, and
+click :guilabel:`New`. From here, the warehouse name can be changed, as well as other details about
+how to resupply the warehouse. Routes can also be set by clicking on the :guilabel:`Routes` smart
+button. Once the warehouse is configured, virtual locations can be created.
+
+.. image:: stock_warehouses/stock-warehouses-create-warehouse.png
    :align: center
+   :alt: The edit screen for creating a new warehouse.
 
-.. note::
-   The *Storage Location* feature will be automatically enabled. Good news, because you will need it
-   later in the process.
+Create a virtual parent location
+--------------------------------
+
+.. important::
+   In order to take stock from multiple warehouses to fulfill a sales order, there need to be at
+   least **two** warehouses acting as children locations of the virtual parent location warehouse.
 
-Now, you have to make sure that the main stock locations of warehouse A and warehouse B are children
-locations of the main stock location of warehouse A + B. Go to the locations menu, and edit the main
-location of your two warehouses. Then, change their parent location to main location of warehouse
-A+B.
+To create and edit locations, go to :menuselection:`Inventory --> Configuration --> Locations`. All
+locations are listed here,including the stock location of the virtual warehouse that was created.
+Click into the stock location for the virtual warehouse created previously, and under the
+**Additional Information** tab, change the :guilabel:`Location Type` from *Internal Location* to
+*View*. This identifies this location as a virtual one used to create a hierarchical structure for
+a warehouse, aggregating its child locations. Products can *not* be stored in this
+:guilabel:`Location Type`.
```

Comment:
Click into the stock location for the virtual warehouse that was previously created. Then, under the
::guilabel::`Additional Information` tab, change the :guilabel:`Location Type` from *Internal Location* to
*View*. 

This identifies this location as a "Virtual" one, used to create a hierarchical structure for
a warehouse, aggregating its "Child" locations. 

.. note::
   Products can *not* be stored in this :guilabel:`Location Type`.